### PR TITLE
Fix not being able to lose in singleplayer

### PR DESCRIPTION
**Pull request by V-rtualized**

### CODE REVIEW

File: Multiplayer/UI/Game_UI.lua [hunk 1]
```diff
@@ -797,15 +797,19 @@ end
...
-	update_new_round_ref(self, dt)
+		update_new_round_ref(self, dt)
 
-	-- Reset ante number
-	G.GAME.win_ante = 8
+		-- Reset ante number
+		G.GAME.win_ante = 8
+		return
+	end
+	update_new_round_ref(self, dt)
```
Comment (TGMM) [inline Multiplayer/UI/Game_UI.lua:812]: Shouldn't we put this in an else so that this function is only called once?

**Review by TGMM — CHANGES_REQUESTED** [review]

File: Multiplayer/UI/Game_UI.lua (hunk 1 above)
Comment (V-rtualized) [reply Multiplayer/UI/Game_UI.lua:812]: Yeah

Comment (V-rtualized) [reply Multiplayer/UI/Game_UI.lua:812]: It has a return in the if statement so it should be fine but I can convert to if-else if you prefer

Comment (TGMM) [reply Multiplayer/UI/Game_UI.lua:812]: Sorry, I'm blind. No leave it as is

**Review by TGMM — APPROVED** [review]

**Pull request merged**